doc/nrf/ug_gzll.rst

```diff
@@ -108,7 +101,7 @@ Gazell automatically notifies the application when a packet is received.
 
 To set up a Gazell application, do the following:
```

Comment:
Shouldn't this `complete the following steps` be a step-based procedure?
Also, would it make sense to split this section into two sections, one for Host, one for Device, given the optional steps for each in the procedure?
Finally, would it make sense to detach disabling scenario from the setup scenario?

Reply:
Okay I have implemented these changes.